client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -491,13 +491,19 @@ class EmailProvidersComparison extends React.Component {
 	}
 
 	handleBack = () => {
-		const { selectedSite } = this.props;
+		const { selectedSite, handleBack } = this.props;
```

Comment:
I think this prop should be called something more like `backHref` or `backPath`, as it's a thing/object. The current name `handleBack` suggests that it will "handle back", making it sound like it should be a function that handles an action.

Reply:
Renamed :)